feature-utils/poly-analysis/src/analysis/report/report-stories.js

```diff
@@ -0,0 +1,21 @@
+export default class ReportStories {
+    constructor(activeReportStories) {
+        this._activeReportStories = activeReportStories;
+    }
+
+    get activeStories() {
+        return this._activeReportStories;
+    }
+
+    get active() {
+        return this.activeStories.length > 0;
```

Comment:
How I understand the code, `ReportStories` is pretending to be a `Story` (see my remark on duck typing above), so that's why it provides an `active` property.

I would suggest a simplification though, I don't see the need for dynamic getters here, except maybe for `jsonReport` (so we don't keep it in memory before it's needed). Could just:

```
class ReportStories {
    constructor(activeReportStories) {
        this.activeStories = activeReportStories;
        this.active = activeReportStories.length > 0;
    }

    get jsonReport() {
        ...
    }
}
```